AdobeStockImageAdminUi/Test/Mftf/Suite/adobe-stock-integration-configuration.xml

```diff
@@ -16,6 +16,10 @@
             </actionGroup>
         </before>
         <after>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminAdobeStockDisableActionGroup" stepKey="disableAdobeStock"/>
+            <actionGroup ref="AdminMediaGalleryEnhancedEnableActionGroup" stepKey="disableEnhancedMediaGallery"/>
+            <actionGroup ref="logout" stepKey="logout"/>
```

Comment:
Is it necessary to logout considering that the next test is going to be executed in a separate browser window?